crates/iota/src/name_commands.rs

```diff
@@ -971,6 +1000,7 @@ impl SubdomainCommand {
 pub enum NameCommandResult {
     Auction(IotaClientCommandResult),
     AuctionMetadata(Auction),
+    Availability((String, Option<u64>)),
```

Comment:
extra tuple should not be needed, though personally I would use struct format anyway

Reply:
Changed